tests/PHPUnit/Unit/DataAccess/LogQueryBuilder/JoinGeneratorTest.php

```diff
@@ -260,13 +260,13 @@ public function test_sortTablesForJoin_anotherTestMakingSureWorksOhPhp5_5()
             'log_link_visit_action',
             array (
                 'table' => 'log_action',
-                'tableAlias' => 'log_action_visit_exit_idaction_name',
-                'joinOn' => 'log_visit.visit_exit_idaction_name = log_action_visit_exit_idaction_name.idaction',
+                'tableAlias' => 'log_action_idaction_name',
+                'joinOn' => 'log_link_visit_action.idaction_name = log_action_idaction_name.idaction',
             ),
             array (
                 'table' => 'log_action',
-                'tableAlias' => 'log_action_idaction_name',
-                'joinOn' => 'log_link_visit_action.idaction_name = log_action_idaction_name.idaction',
+                'tableAlias' => 'log_action_visit_exit_idaction_name',
+                'joinOn' => 'log_visit.visit_exit_idaction_name = log_action_visit_exit_idaction_name.idaction',
```

Comment:
FYI: This test is actually slightly buggy because the query would never work as it references a `log_visit` table which is never added.  I presume it needs to be `log_link_visit_action` instead.